ui/app/serializers/clients/activity.js

```diff
@@ -43,6 +45,7 @@ export default class ActivitySerializer extends ApplicationSerializer {
         non_entity_clients,
       };
     }
+    return object;
```

Comment:
Right now this returned object will look like this (based on RFC draft) - TODO is if we want to delete unnecessary keys

```     
         { label:"ns1/",
            distinct_entities:5,
            non_entity_tokens:5,
            clients:10,
            entity_clients:5,
            non_entity_clients:5
         }